pkg/sql/sem/tree/function_definition.go

```diff
@@ -28,6 +28,30 @@ type FunctionDefinition struct {
 	FunctionProperties
 }
 
+// Volatility indicates whether the result of a function is dependent *only*
+// on the values of its explicit arguments, or can change due to outside factors
+// (such as parameter variables or table contents).
+//
+// Each value here must be ordered such that the property i+1 is "weaker" the property
```

Comment:
Why does the order matter? I don't see any code that cares about the relative ordering of this.

Reply:
ah, i was planning to actually use it then decided not to.. resolved.